### PR TITLE
[T2A2][T11-A4]Evan Yeung

### PR DESCRIPTION
**Pull request by evanyeung**

### CODE REVIEW

**Comment by evanyeung** [comment]
Ready for review

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -44,7 +44,7 @@
     /**
      * Version info of the program.
      */
-    private static final String VERSION = "AddessBook Level 1 - Version 1.0";
+    private static final String VERSION = "AddressBook Level 1 - Version 1.0";
```
Comment (ZawadulFarhan) [inline src/seedu/addressbook/AddressBook.java:47]: Good catch!

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -339,8 +338,7 @@ private static boolean hasValidParentDirectory(Path filePath) {
      * If a file already exists, it must be a regular file.
      */
     private static boolean hasValidFileName(Path filePath) {
-        return filePath.getFileName().toString().lastIndexOf('.') > 0
-                && (!Files.exists(filePath) || Files.isRegularFile(filePath));
+        return fileHasExtension(filePath) && isRegularFile(filePath);
```
Comment (ZawadulFarhan) [inline src/seedu/addressbook/AddressBook.java:341]: Good use of abstraction! Name could be improved. hasExtension(filePath) would be a better choice. Have a look at the naming conventions for boolean function in the conding standards document.

**Review by ZawadulFarhan — COMMENTED** [review]
Good job on the activity!

File: src/seedu/addressbook/AddressBook.java [hunk 3]
```diff
@@ -394,8 +392,10 @@ private static String executeCommand(String userInputString) {
      * @return  size 2 array; first element is the command type and second element is the arguments string
      */
     private static String[] splitCommandWordAndArgs(String rawUserInput) {
-        final String[] split =  rawUserInput.trim().split("\\s+", 2);
-        return split.length == 2 ? split : new String[] { split[0] , "" }; // else case: no parameters
+        final String[] splitUserInput =  rawUserInput.trim().split("\\s+", 2);
+        return splitUserInput.length == 2
+                ? splitUserInput
+                : new String[] { splitUserInput[0] , "" }; // else case: no parameters
```
Comment (ZawadulFarhan) [inline src/seedu/addressbook/AddressBook.java:398]: Good use of multiline statement. I believe the spaces are not properly done for new String[] {}.

**Pull request closed without merging**